README.md

```diff
@@ -1 +1,24 @@
-# Hello-Microverse
\ No newline at end of file
+Project Name
```

Comment:
- [ ] Your README file is almost there, great job! Let's make it even better by adding some more information, this way your repo will look more professional, try adding these sections: 

- Contributing
- Show your support
- Acknowledgments
- License

(Here's a [template](https://github.com/microverseinc/readme-template) you can use)

Reply:
Good morning sir @smunozmo 
I applied the changes based on your request
please help to review it again

Thanks